### ISSUE TITLE
Export incomplete: missing _layouts folder

### ISSUE THREAD

**Issue by maxx37** [Typesetter/Typesetter#628]
Hello,

I'm using the 5.1 version. When I make an export to move a site to another server, it appears that the /data/_layouts folder is not included in the zip file. Although I left the include options unchanged, so normally layouts should be in there. Also, when I try to import the zip file on the other server, it is refused (probably for the reason mentioned above).

Beste regards, Marc

**Comment by juek** [comment]
Already fixed with https://github.com/Typesetter/Typesetter/commit/bd56fbac71c1989274b1ebfeef70aa300ecb6990

**Comment by juek** [comment]
> Also, when I try to import the zip file on the other server, it is refused (probably for the reason mentioned above).

What exactly do you mean by 'refused'?

**Comment by maxx37** [comment]
Thank you for your quick response.
If I understand well, this is already fixed since October 2017.
But the version that I downloaded no longer ago than last week from typesettercms.com still dates from 12-08-2017, according to the file dates and times.
I did not notice it was so old, maybe it's a good idea to replace it also there with a more recent one...

I will try with the master version from Github, maybe the refused restore file is not happening anymore.

Best regards, Marc
